### PR TITLE
Backport-2.5-1495 Adds performance tuning chapter to 2.5 EDA controller user guide

### PR DESCRIPTION
**Pull request by jself-sudoku**
Backporting new performance tuning chapter to the EDA controller user guide.

Copied from original PR: 

- Files added:

     modules/eda/con-modifying-simultaneous-activations.adoc
     modules/eda/con-modifying-memory-limit.adoc
     modules/eda/con-system-level-monitoring.adoc
     modules/eda/proc-modifying-activations-after-install.adoc
     modules/eda/proc-modifying-activations-during-install.adoc
     modules/eda/proc-modifying-memory-after-install.adoc
     modules/eda/proc-modifying-memory-during-install.adoc
     modules/eda/ref-performance-troubleshooting.adoc
     assemblies/eda/assembly-eda-performance-tuning.adoc

- Files modified:

     titles/eda/eda-user-guide/master.adoc

### CODE REVIEW

File: downstream/modules/eda/con-modifying-simultaneous-activations.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+= Modifying the number of simultaneous rulebook activations 
+
+[role="_abstract"]
+By default, {EDAcontroller} allows 12 rulebook activations to run simultaneously.
+If any more than 12 rulebook activations are created, the expected behavior is that subsequent rulebook activations wait in pending until there is an available rulebook activation worker.
+In this case, the rulebook activation status will display as “pending” even if there is enough free memory and CPU on your {EDAcontroller} instance.
+To change this behavior, you must change the default maximum number of running rulebook activations.
+
+**Note:** The value for `EDA_MAX_RUNNING_ACTIVATIONS` doesn’t change with the change in instance size, so it needs to be adjusted manually.
```
Comment (hherbly) [inline downstream/modules/eda/con-modifying-simultaneous-activations.adoc:11]: I would follow the note syntax shown [here](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions)--this is how I see the majority of our notes formatted.

**Review by hherbly — APPROVED** [review]
lgtm

**Comment by jself-sudoku** [comment]
Thanks, @hherbly . I'll update the syntax for the note when I update two sections of the performance tuning chapter for 2.5.

File: downstream/modules/eda/con-modifying-simultaneous-activations.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/modules/eda/con-modifying-simultaneous-activations.adoc:11]: Will apply this change to the content when I update the other sections that are changing for 2.5.

**Pull request merged**